install-dev/upgrade/php/ps_1780_add_feature_flag_tab.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+function ps_1780_add_feature_flag_tab()
+{
+    $className = 'AdminFeatureFlag';
+
+    $result = Db::getInstance()->executeS(
+        'SELECT id_tab FROM `'._DB_PREFIX_.'tab` WHERE `class_name` = \'AdminAdvancedParameters\''
+    );
+
+    if (empty($result)) {
+        return;
+    }
+    $advancedParametersTabId = (int) $result[0]['id_tab'];
+
+    include_once _PS_INSTALL_PATH_.'upgrade/php/add_new_tab.php';
```

Comment:
Check the file existence

Reply:
So you mean "in case _PS_INSTALL_PATH_ is not correctly set" ?

Reply:
Not useful, it's one of our usual tools for upgrade It must be present, period

Reply:
It should be present :stuck_out_tongue_closed_eyes:

Reply:
<img src="https://pbs.twimg.com/media/EwhNXtLXIAAnIVJ?format=jpg&name=small">